R/git.R

```diff
@@ -0,0 +1,22 @@
+#' Git information
+#' 
+#' Displays git information from the repository
+#' 
+#' @import git2r
+#' @import shiny
+#' @export
+git_info <- function() {
+  print(getwd())
```

Comment:
I don't think we should leave `print` statements like that. It is side effect. Let developer choose, maybe he will read `git_info()` and then prints it somewhere.